azure/Guidelines.md

```diff
@@ -907,45 +903,59 @@ For a PUT (create or replace) with additional long-running processing:
 
 <a href="#lro-put-returns-operation-location" name="lro-put-returns-operation-location">:ballot_box_with_check:</a> **YOU SHOULD** include an `Operation-Location` header in the response with the absolute URL of the status monitor for the operation.
 
-<a href="#lro-put-operation-location-includes-api-version" name="lro-put-operation-location-includes-api-version">:ballot_box_with_check:</a> **YOU SHOULD** include the `api-version` query parameter in the `Operation-Location` header with the same version passed on the initial request if it is required by the get operation on the status monitor.
+<a href="#lro-put-operation-location-includes-api-version" name="lro-operation-location-includes-api-version">:ballot_box_with_check:</a> **YOU SHOULD** include the `api-version` query parameter in the `Operation-Location` header with the same version passed on the initial request but expect a client to change the api-version value to whatever a new/different client desires it to be.
```

Comment:
This change isn't necessary because lro-status-monitor-accepts-any-api-version already states that the GET on the status monitor must accept any api-version.